build.sbt

```diff
@@ -49,6 +53,8 @@ libraryDependencies ++= Seq(
   "eus.ixa" % "ixa-pipe-nerc" % "1.6.1"
 )
 
+libraryDependencies += "nz.ac.waikato.cms.weka" % "optics_dbScan" % "1.0.4"
```

Comment:
DBSCAN is not trivially parallizable so I'm not surprised that the in-proc implementations rule for now :)